packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/[notebookId]/dashboard/_components/Resources.component.tsx

```diff
@@ -0,0 +1,152 @@
+import { Cpu, HardDrive, HelpCircle, MemoryStick, Zap } from 'lucide-react';
+import { useTranslation } from 'react-i18next';
+import { useNotebookData } from '../../Notebook.context';
+import { bytesConverter } from '@/lib/bytesHelper';
+import { Slider } from '@/components/ui/slider';
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from '@/components/ui/popover';
+
+const Resources = () => {
+  const { notebook } = useNotebookData();
+  const { t } = useTranslation('pci-ai-notebooks/notebooks/notebook/dashboard');
+  return (
+    <div data-testid="resources-container">
+      {notebook.spec.resources.gpu > 0 ? (
+        <div className="flex flex-col">
+          <div className="flex flex-row gap-2 items-center">
+            <h5>Power</h5>
```

Comment:
Do we display "Power" for every language?

Reply:
Fixed